### PR TITLE
Adding IconButton to generate names for wallet names for new_wallet a…

### PR DESCRIPTION
**Pull request by RafiaChy**
1. used suffixIcon to add an IconButton
2. Icon used is refresh.png
3. IconButton tied to generateName function
4.  On clicking IconButton, the wallet name fields of  both new-wallet and restore-wallet gets pasted with generated names

### CODE REVIEW

File: lib/entities/generate_name.dart [hunk 1]
```diff
@@ -20,5 +20,6 @@ Future<String> generateName() async {
   final chosenNoun = nouns[randomThing.nextInt(nouns.length)];
   final returnString =
       chosenAdjective.capitalized() + ' ' + chosenNoun.capitalized();
+
```
Comment (mkyq) [inline lib/entities/generate_name.dart:23]: Why it appeared here ? Do not modify parts which are not effected by your implementation

File: lib/src/screens/new_wallet/new_wallet_page.dart [hunk 2]
```diff
@@ -92,44 +98,76 @@ class _WalletNameFormState extends State<WalletNameForm> {
...
+                  style: TextStyle(
+                      fontSize: 20.0,
+                      fontWeight: FontWeight.w600,
+                      color: Theme.of(context).primaryTextTheme.title.color),
+                  decoration: InputDecoration(
+                    suffixIcon: IconButton(
+                      onPressed: () async {
+                        final String rName = await generateName();
+
+                        print(rName);
```
Comment (mkyq) [inline lib/src/screens/new_wallet/new_wallet_page.dart:121]: Remove it, as it doesn't require by your implementation

File: lib/src/screens/new_wallet/new_wallet_page.dart [hunk 3]
```diff
@@ -92,44 +98,76 @@ class _WalletNameFormState extends State<WalletNameForm> {
...
+                  controller: _controller,
+                  textAlign: TextAlign.center,
+                  style: TextStyle(
+                      fontSize: 20.0,
+                      fontWeight: FontWeight.w600,
+                      color: Theme.of(context).primaryTextTheme.title.color),
+                  decoration: InputDecoration(
+                    suffixIcon: IconButton(
+                      onPressed: () async {
+                        final String rName = await generateName();
```
Comment (mkyq) [inline lib/src/screens/new_wallet/new_wallet_page.dart:119]: Do not define type for `final` variables

File: lib/src/screens/restore/wallet_restore_from_seed_form.dart [hunk 4]
```diff
@@ -57,9 +58,38 @@ class WalletRestoreFromSeedFormState extends State<WalletRestoreFromSeedForm> {
...
+            validator: WalletNameValidator(),
+            suffixIcon: Container(
+              width: 12,
+              height: 14,
+              margin: const EdgeInsets.only( bottom: 15, left: 13),
+              child: InkWell(
+                onTap: () async {
+                  final String rName = await generateName();
+
+                  print(rName);
```
Comment (mkyq) [inline lib/src/screens/restore/wallet_restore_from_seed_form.dart:72]: Same here. Do not define types for `final`s. And remove unnecessary prints.

**Review by mkyq — CHANGES_REQUESTED** [review]

File: lib/src/screens/new_wallet/new_wallet_page.dart (hunk 2 above)
Comment (RafiaChy) [reply lib/src/screens/new_wallet/new_wallet_page.dart:121]: @mkyq removed print statement

File: lib/src/screens/new_wallet/new_wallet_page.dart (hunk 3 above)
Comment (RafiaChy) [reply lib/src/screens/new_wallet/new_wallet_page.dart:119]: @mkyq removed the type

**Comment by RafiaChy** [comment]
@mkyq , I have added a generate-name button for restore-from-key page and 

1. removed empty lines from generate_name.dart, 
2. types for final variables in new_wallet.page.dart and restore_from_seed_form.dart, and 
3. print statements in new_wallet.page.dart and restore_from_seed_form.dart, and

**Review by mkyq — APPROVED** [review]

**Pull request merged**